mempool/src/thread_pool.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+use once_cell::sync::Lazy;
+
+pub(crate) static IO_POOL: Lazy<rayon::ThreadPool> = Lazy::new(|| {
```

Comment:
Will having a static global threadpool cause any issue for unit tests that run in parallel?

Reply:
I don't see why it would.

Reply:
Hmm.. the concern is one bad test hogging the IO pool can make other tests flaky as well I guess? Is there any reason this needs to be static global? Can we not make it one thread pool per mempool?

Reply:
Most rayon calls in the repo are using the global pool anyway?